services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -285,6 +290,7 @@ async def retrieve_service_inputs(
 
         return RetrieveDataOutEnveloped.from_transferred_bytes(transferred_bytes)
 
+    # NOTE: used by API indirectly
```

Comment:
why all these comments? also be aware that ```isort``` sometimes moves these at the top of the file... so I'm not sure it's such a great idea

Reply:
Let me just drop them. If they cause issues.